charts/hono/templates/hono-service-device-registry-base/headless-svc.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.deviceRegistryExample.enabled }}
+{{- if and .Values.deviceRegistryExample.enabled ( eq .Values.deviceRegistryExample.type "file" ) }}
```

Comment:
Since it is only used for the file based registry, this file should probably be moved to `hono-service-device-registry-file`, shouldn't it?